remotespark/utils/sparkevents.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+import importlib
+import remotespark.utils.configuration as conf
+import remotespark.utils.constants as constants
+
+
+class SparkEvents:
+    def __init__(self):
+        """
+        Create an instance from the handler mentioned in the config file.
+        """
+        module, class_name = conf.events_handler_class().rsplit('.', 1)
+        events_handler_module = importlib.import_module(module)
+        events_handler = getattr(events_handler_module, class_name)
+
+        self.handler = events_handler()
+
+
+    def emit_session_creation_start_event(self, session_guid, language):
+        """
+        Emitting Start Session Event
+        """
+        assert language in constants.SESSION_KINDS_SUPPORTED
+
+        event_name = constants.SESSION_CREATION_START_EVENT
+        time_stamp = datetime.now()
```

Comment:
Getting the current time should be in a method by itself. What is the timezone used by `datetime.now()`?